deploy/chart/templates/deployment.yaml

```diff
@@ -16,25 +16,35 @@ spec:
         app.kubernetes.io/name: {{ include "local-path-provisioner.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
-    {{- with .Values.imagePullSecrets }}
+    {{- if .Values.defaultSettings.registrySecret }}
```

Comment:
`.Values.defaultSettings.registrySecret` should only be used in the ServiceAccount. 
So changes from line 19 to 21 should be discarded.

Reply:
Hi, I´ve reverted the change.